lvm-driver.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: lvm-driver
+  version: 1.6.2
+  epoch: 0
+  description: Dynamically provision Stateful Persistent Node-Local Volumes & Filesystems for Kubernetes that is integrated with a backend LVM2 data storage stack.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - btrfs-progs
+      - busybox
+      - ca-certificates-bundle
+      - device-mapper
+      - e2fsprogs
+      - e2fsprogs-extra
+      - lvm2
+      - util-linux
+      - xfsprogs
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/openebs/lvm-localpv
+      tag: lvm-localpv-${{package.version}}
+      expected-commit: 45e0fbdd6af38f3025132fff5b6d10c3c2eec1fb
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.23.0 google.golang.org/grpc@v1.56.3 google.golang.org/protobuf@v1.33.0 github.com/golang/protobuf@v1.5.4
+
+  - uses: go/build
+    with:
+      packages: ./cmd/main.go
+      ldflags: |
+        -X github.com/openebs/lvm-localpv/pkg/version.Version=${{package.version}}
+        -X github.com/openebs/lvm-localpv/pkg/version.GitCommit=$(git rev-parse HEAD)
+      output: lvm-driver
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "upstream image have executable placed at /usr/local/bin"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/usr/local/bin/
+          ln -sf /usr/bin/lvm-driver ${{targets.contextdir}}/usr/local/bin/lvm-driver
+
+update:
+  enabled: true
+  github:
+    identifier: openebs/lvm-localpv
+    strip-prefix: lvm-localpv-
+
+test:
+  pipeline:
+    - runs: |
+        lvm-driver --version version 2>&1 | grep -q ${{package.version}}
```

Comment:
Is it possible to expand the tests a bit?

Reply:
I can do that @kranurag7